# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1232463729 18000
#      Tue Jan 20 10:02:09 2009 -0500
# Node ID 1eb16d930f3ccd7fb8d165ce5ca7581a2c41b4e3
# Parent  67a632417879460565f4734a18afc1d34a2d6dc2
package.txi: @DOCSTRING for pkg

diff --git a/doc/interpreter/package.txi b/doc/interpreter/package.txi
--- a/doc/interpreter/package.txi
+++ b/doc/interpreter/package.txi
@@ -102,16 +102,18 @@ command
 pkg install -nodeps my_package_with_dependencies.tar.gz
 @end example
 
 @noindent
 Since the installed package expects its dependencies to be installed
 it may not function correctly. Because of this it is not recommended
 to disable dependency checking.
 
+@DOCSTRING(pkg)
+
 @node Using Packages
 @section Using Packages
 
 By default installed packages are available from the Octave prompt,
 but it is possible to control this using the @code{pkg load} and 
 @code{pkg unload} commands. The functions from a package can be 
 removed from the Octave path by typing
 
